docs/pages/admin-guides/management/diagnostics/metrics.mdx

```diff
@@ -14,12 +23,168 @@ collect from your Teleport cluster.
 This will enable the `http://127.0.0.1:3000/metrics` endpoint, which serves the
 metrics that Teleport tracks. It is compatible with [Prometheus](https://prometheus.io/) collectors.
 
-The following metrics are available:
+## Backend operations
+
+A Teleport cluster cannot function if the Auth Service does not have a healthy
+cluster state backend. You need to track the ability of the Auth Service to read
+from and write to its backend. 
+
+The Auth Service can connect to [several possible
+backends](../../../reference/backends.mdx). In addition to Teleport backend
+metrics, you should set up monitoring for your backend of choice so that, if
+these metrics show problematic values, you can correlate them with metrics on
+your backend infrastructure.
+
+### Backend operation throughput and availability
+
+On each backend operation, the Auth Service increments a metric. Backend
+operation metrics have the following format:
+
+```text
+teleport_backend_<METRIC_NAME>[_failed]_total
+```
+
+If an operation results in an error, the Auth Service adds the `_failed` segment
+to the metric name. For example, successfully creating a record increments the
+`teleport_backend_write_requests_total` metric. If the create operation fails,
+the Auth Service increments `teleport_backend_write_requests_failed_total`
```

Comment:
As an aside, since I'm often looking for useful metrics as an operator, is there a reason this metric doesn't get surfaced [here](https://goteleport.com/docs/reference/monitoring/metrics/)?  I would expect this to be the full list of metrics surfaced by Teleport.  I've typically had to go spelunking in [metrics.go](https://github.com/gravitational/teleport/blob/e1abc28dcdfabe9010dba4df2ba71d30fbe441a6/metrics.go#L187) to find what I'm looking for since it includes more than that page.

Reply:
We have added to the metrics reference incrementally without any systematic checks for accuracy and completeness. We should auto-generate the reference. I've opened an issue: https://github.com/gravitational/teleport/issues/47000